src/packagedcode/models.py

```diff
@@ -389,6 +389,11 @@ class Package(BasePackage):
         label='keywords',
         help='A list of keywords.')
 
+    classifiers = List(
```

Comment:
Can you elaborate what you are trying to achieve here? This seems to conflict with the standard Package URL mapping used for classifiers in https://github.com/nexB/scancode-toolkit/pull/1960/files#diff-9feddae6239d49e50571ae6742c91612L223